src/palette/internal/palettecell.cpp

```diff
@@ -224,6 +230,41 @@ bool PaletteCell::read(XmlReader& e)
     return add && element;
 }
 
+void PaletteCell::migrateOldItemIfNeeded()
```

Comment:
I think it's better to add a separate class with name like "PaletteCompat" and add this function to it as static with input arg and output (new item);
In the future, we may add other fefunctions to this class and we will see what kind of compatibility we support.

Reply:
@igorkorsukov done! 👍